docs/source/_developers/primer.md

```diff
@@ -0,0 +1,234 @@
+
+##  Images and Diagrams
+Visual content like images and diagrams can significantly enhance your documentation. Here are some guidelines:
+- **High-Quality Images**: Always use high-resolution images that are clear and easy to see. If you're using screenshots, ensure they accurately represent the current version of your product or interface.
+- **Descriptive Alt Text**: Always provide alt text for your images. The alt text describes the image for people who can't see it, whether due to visual impairments, technical issues, or because they're using a screen reader.
+- **Simple Diagrams**: Keep your diagrams simple. Avoid including unnecessary details that could confuse your readers.
+- **Consistent Style**: If you use multiple diagrams, ensure they have a consistent style and use the same symbols and conventions.
+
+## Markdown and MyST Syntax
+
+Markdown and MyST provide a simple way to format your content. Here are some tips for using these syntax effectively:
+- **Basic Markdown**: Familiarize yourself with basic markdown syntax, including headers, lists, links, images, and code blocks.
+- **MyST Features**: MyST extends the standard markdown syntax with additional features such as cross-references, footnotes, directives, warnings, and other particular elements. Make sure you understand how to use these features correctly.
+- **Code Blocks**: Use triple backticks to create code blocks and specify the language for syntax highlighting.
+- **Admonitions**: Use MyST's admonition syntax to highlight important information. For example, `:::{note}` creates a note, and `:::{warning}` creates a warning.
+
+:::{note}
+There are many [admonitions] to choose from. Use your best judgement, look at other examples, and be sure to use the appropriate tone.
+:::
+
+- **Consistency**: Be consistent with your use of MyST syntax. For example, always use the same header level for section titles, the same style for lists, etc.
+- Follow the MyST syntax for advanced features such as directives and roles.
+
+##  Review Process
+Finally, ensure your documentation undergoes a thorough review process before it's published. We do this through PRs on GitHub (see {ref}`git-primer`).
+
+- **Peer Review**: Encourage team members to review each other's work. This can help catch errors, ambiguities, and inconsistencies the original writer may have missed.
+- **Proofreading**: Check for spelling, grammar, and punctuation errors. Tools like Grammarly or language-tool can help with this.
+
+
+:::{tip}
+When add a substantial amount of content, whether a new page or several sections, consider using Notion to draft. Then, update the RTD pages (i.e., this project) for a technical review. This way, development can be more interactive, time is not wasted adjusting formatting and other syntax, and the history is much simpler (and more promising) than that of our own managed via Git.
```

Comment:
I really do like this suggestion.

Reply:
When add a substantial amount of content -> When adding a substantial amount of content